apis/v1alpha1/instrumentation_types.go

```diff
@@ -36,6 +37,10 @@ type InstrumentationSpec struct {
 	// +optional
 	Sampler `json:"sampler,omitempty"`
 
+	// Env defines env vars.
```

Comment:
We should document precedence order. Does the env var here override env var in spec.java.env or is it vice versa? Does the env var override setting like `spec.exporter.endpoint`?